03.asynchronous/fetch.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
`lang="ja"` かなと思います。

```suggestion
<html lang="ja">
```